masterlist.yaml

```
@@ -28851,7 +28858,7 @@ plugins:
     url: [ 'https://www.nexusmods.com/oblivion/mods/10671/' ]
     inc:
```

```suggestion
    inc:
      - *Cobl
```

Aliases should be sorted above non-aliases for consistency.